src/main/scala/encry/cli/commands/CreateAccountFromSeed.scala

```diff
@@ -0,0 +1,23 @@
+package encry.cli.commands
+
+import akka.util.Timeout
+import akka.pattern._
+import encry.cli.{Ast, Response}
+import encry.settings.EncryAppSettings
+import encry.EncryApp._
+import encry.view.NodeViewHolder.ReceivableMessages.{CreateAccountManagerFromSeed, NewAccountManager}
+
+import scala.concurrent.Future
+
+object CreateAccountFromSeed extends Command {
```

Comment:
add description of this command in "app help", also add example of command usage(ex RemoveFromBlackList:9)